app/code/Magento/Backend/Controller/Adminhtml/Dashboard/Graph/Amounts.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Backend\Controller\Adminhtml\Dashboard\Graph;
+
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Controller\Adminhtml\Dashboard;
+use Magento\Backend\Model\Dashboard\Graph;
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Framework\Controller\Result\Json;
+use Magento\Framework\Controller\ResultFactory;
+
+class Amounts extends Dashboard implements HttpPostActionInterface
+{
+    /**
+     * @var Graph
+     */
+    private $graph;
+
+    /**
+     * Amounts constructor.
+     *
+     * @param Context $context
+     * @param Graph $graph
+     */
+    public function __construct(
+        Context $context,
+        Graph $graph
+    ) {
+        parent::__construct($context);
+        $this->graph = $graph;
+    }
+
+    /**
+     * @return Json
+     */
+    public function execute(): Json
+    {
+        $data = [
+            'data' => $this->graph->getByPeriod($this->_request->getParam('period'), 'revenue'),
+            'label' => __('Revenue')
+        ];
+
+        /** @var Json $result */
+        $result = $this->resultFactory->create(ResultFactory::TYPE_JSON);
```

Comment:
Could you add specific factory to constructor and use it there?
There were long discussion for this some time ago, you can see more details here:
https://github.com/magento/magento2/pull/24711#issuecomment-542469839

Reply:
Done @ihor-sviziev